.github/workflows/publish.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           # manylinux docker image, but I haven't figured out how to install CUDA on manylinux.
           os: [ubuntu-20.04]
           python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
-          torch-version: ['1.12.1', '1.13.1', '2.0.1', '2.1.1', '2.2.0.dev20231130']
+          torch-version: ['1.12.1', '1.13.1', '2.0.1', '2.1.2', '2.2.0.dev20231213', '2.3.0.dev20240126']
```

Comment:
Let's wait a couple of days to see which torch nightly version that nvcr 24.01 uses :D

Reply:
Revert the `2.1.2` and `2.2.0.dev20231213` changes, keep `2.3.0.dev20240126`?

Reply:
Let's do 2.1.2, [whichever version that nvcr 24.01 will use], and 2.3.0.dev20240126.

Reply:
okay, I won't change nightly dev tag for torch 2.2 in this PR